doc/learning/tutorial_todoapp.md

```diff
@@ -502,6 +502,10 @@ deleteTask(task) {
 Notice that the `onDelete` prop is defined with a `.bind` suffix: this is a special
 suffix that makes sure the function callback is bound to the component.
 
+Notice also that we have two functions named `deleteTask`. The one in the Task
+component just delegates the work to the Root component that owns the task list
+via the `onDelete' property.
```

Comment:
Suggestion: Wrap the onDelete with 2 backticks.

\`onDelete' -> \`onDelete\`

Reply:
@caburj Sorry for the typo! I amended the commit to correct it.

Reply:
@NL66278 One more thing. The commit title, can you change it to "[DOC] Add a notice to step 9". Then we can merge. :)